### ISSUE TITLE
Add jquery 3 support

### ISSUE THREAD

**Issue by inferno-cc** [johnfawcett/dynroute#6]
Hey,

The delete bttn is not more working with freepbx 14. Reason? Updated jquery.

```
jquery-3.1.1.min.js?load_version=14.0.1.4:2 Uncaught TypeError: $(...).live is not a function
    at HTMLDocument.<anonymous> (dynroute.js?load_version=14.0.0:54)
    at j (jquery-3.1.1.min.js?load_version=14.0.1.4:2)
    at k (jquery-3.1.1.min.js?load_version=14.0.1.4:2)
```

The live function is not more used since 1.9.

Please change the code @ '$('.delete_entrie').live' to:

```
	$('body').on('click', '.delete_entrie', function(e){
		e.preventDefault();
		$(this).closest('tr').fadeOut('normal', function(){$(this).closest('tr').remove();});
	});
```

**Comment by inferno-cc** [comment]
#7

**Comment by johnfawcett** [comment]
Thanks for pointing out this issue and for submitting a suggested fix. I checked into ivr module (from which the code was derived) and saw the following commit that addressed this issue:

https://git.freepbx.org/projects/FREEPBX/repos/ivr/commits/ee00277c20443ab4c8e8dab0f5547e229e1a4e3b#assets/js/ivr.js

I have added a fix to dynamic routes based on that and your suggested fix. There is only a slight difference compared to the one submitted (document instead of body). I noticed that the ivr module prevents deleting the first or last entry. I am not sure why that is, but I did not import that part of the code, since it didn't seem to work well with dynamic routes. Released version 14.0.1.
